lib/config.js

```diff
@@ -302,9 +302,12 @@ config.openTelemetryEnabled = false;
  * Note that the `console` exporter should almost definitely NEVER be used in
  * production environments.
  *
- * @type {'console' | 'honeycomb'}
+ * @type {'console' | 'honeycomb' | 'jaeger'}
```

Comment:
TIL about Jaeger!

Reply:
Yeah, it's really nice for locally exploring traces without needing to export them all the way to Honeycomb! We've supported it since #6514, this type just needed to be updated.